app/experimenter/nimbus-ui/src/components/FormOverview/index.tsx

```diff
@@ -163,6 +181,52 @@ const FormOverview = ({
           </Form.Text>
           <FormErrors name="hypothesis" />
         </Form.Group>
+
+        {experiment && (
+          <Form.Group controlId="riskBrand" as={Container}>
+            <Row className="align-items-center mt-4 mb-4">
+              <Col sm={10} className="col-sm-10 pl-0">
+                <Form.Label className="mb-0">
+                  {isMissingField!("risk_brand") && (
+                    <span className="align-text-bottom mr-1">
+                      <InlineErrorIcon
+                        name="risk_brand"
+                        message="Please answer this question"
+                      />
+                    </span>
+                  )}
```

Comment:
Nit: In other fields, this icon shows up over to the right at the end of a label or input field. Should this follow that pattern?